CMakeLists.txt

```diff
@@ -668,6 +668,17 @@ if(BUILD_TOOLS AND BUILD_DX11 AND WIN32)
   set_property(DIRECTORY PROPERTY VS_STARTUP_PROJECT texconv)
 endif()
 
+#--- install .exes
+
+if(MINGW AND BUILD_TOOLS)
```

Comment:
This is not particularly specific to MinGW. That said, I have to handle the EXE publishing for VCPKG in a special way to use the code-signed versions when possible.

Please make this:

```
if(BUILD_TOOLS AND (NOT VCPKG_TOOLCHAIN))
```